recipes/mgkit/meta.yaml

```diff
@@ -13,26 +13,23 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 1
+  number: 0
   skip: True # [not py27]
   entry_points:
-    - download_data = mgkit.workflow.download_data:main
-    - download_profiles = mgkit.workflow.download_profiles:main
     - filter-gff = mgkit.workflow.filter_gff:main
     - add-gff-info = mgkit.workflow.add_gff_info:main
     - get-gff-info = mgkit.workflow.extract_gff_info:main
     - hmmer2gff = mgkit.workflow.hmmer2gff:main
     - blast2gff = mgkit.workflow.blast2gff:main
     - snp_parser = mgkit.workflow.snp_parser:main
-    - translate_seq = mgkit.workflow.nuc2aa:main
     - fastq-utils = mgkit.workflow.fastq_utils:main
     - taxon-utils = mgkit.workflow.taxon_utils:main
     - json2gff = mgkit.workflow.json2gff:main
     - fasta-utils = mgkit.workflow.fasta_utils:main
     - sampling-utils = mgkit.workflow.sampling_utils:main
 
 requirements:
-  host:
+  build:
```

Comment:
This has to be host, not build. See the new conda-build 3 docs. build is only for build tools like compiler or make.